libmscore/trill.cpp

```diff
@@ -474,6 +474,9 @@ bool Trill::setProperty(Pid propertyId, const QVariant& val)
     case Pid::ORNAMENT_STYLE:
         setOrnamentStyle(MScore::OrnamentStyle(val.toInt()));
         break;
+    case Pid::COLOR:
+        setColor(val.value<QColor>());
+        [[fallthrough]];
```

Comment:
I think, better add macro to log.h and use it

Reply:
This is libmscore...